lane.py

```diff
@@ -1409,10 +1664,8 @@ def featured_window(self, target_size):
         :param: A 2-tuple (low value, high value), or None if the
         entire span should be considered.
         """
-        if self.size < 100:
-            # This lane is so small that there's a significant risk
-            # that the interval might not contain enough works. Just
-            # choose randomly from the entire lane.
+        if self.size < target_size:
+            # Don't bother -- we're returning the whole lane.
```

Comment:
Rather than sending the entire interval if the target size is less than a specific number, send the entire interval if the target size is as big or bigger than the lane. If the target size is just really big compared to the lane, the math will work out.